src/controller.rs

```diff
@@ -67,6 +67,9 @@ pub struct RtsCameraControls {
     /// Speed of camera pan (either via keyboard controls or edge panning).
     /// Defaults to `1.0`.
     pub pan_speed: f32,
+    /// How much the camera will zoom (i.e. the camera zoom sensibility).
+    /// Defaults to `0.5`.
+    pub zoom_step: f32,
```

Comment:
A few points:

* I think `zoom_sensitivity` is a better name, because "step" doesn't make sense when zooming with a touchpad, for example
* It's better if the default is `1.0` rather than an arbitrary value, so that it effectively acts as a multiplier of the standard amount. You'll of course need to keep the `0.5` multiplication in the system.
* You misspelled "sensitivity" in the comment

Reply:
Solved!